### PR TITLE
Update to Bot API 5.3

### PR DESCRIPTION
**Pull request by kraftwerk28**
The only thing I omitted there is what was added in #26

### CODE REVIEW

File: proxied.d.ts [hunk 1]
```diff
@@ -669,7 +670,9 @@ export interface Typegram<F> {
       file_id: string;
     }): File;
 
-    /** Use this method to kick a user from a group, a supergroup or a channel. In the case of supergroups and channels, the user will not be able to return to the chat on their own using invite links, etc., unless unbanned first. The bot must be an administrator in the chat for this to work and must have the appropriate admin rights. Returns True on success. */
+    /**
+     * @deprecated
```
Comment (EdJoPaTo) [inline proxied.d.ts]: Maybe highlight what to use instead?
```suggestion
     * @deprecated use banChatMember instead
```

File: proxied.d.ts [hunk 2]
```diff
@@ -861,12 +876,20 @@ export interface Typegram<F> {
       chat_id: number | string;
     }): ChatMember[];
 
-    /** Use this method to get the number of members in a chat. Returns Int on success. */
+    /**
+     * @deprecated
```
Comment (EdJoPaTo) [inline proxied.d.ts]: Maybe highlight what to use instead?
```suggestion
     * @deprecated use getChatMemberCount instead
```

**Comment by KnorpelSenf** [comment]
LGTM on first sight.

I'm gonna do a few tiny tweaks to formatting and style tomorrow because I'm a pedantic weirdo when it comes to typegram, but I don't need to bother you with that 😅

**Comment by kraftwerk28** [comment]
What about husky hook plus prettier to make it automated?

**Comment by KnorpelSenf** [comment]
That's beyond what prettier can do, or what it's meant for. Wait until you see the changes :D

**Comment by KnorpelSenf** [comment]
@kraftwerk28 that should be it. I am going to merge and publish if you confirm I did not make any unresolved mistakes.

**Pull request merged**